openedx/adg/lms/applications/admin.py

```diff
@@ -203,6 +203,15 @@ class UserApplicationADGAdmin(admin.ModelAdmin):
     list_filter = ('status', )
     list_per_page = 10
 
+    def get_queryset(self, request):
+        """
+        Override `get_queryset` method of BaseModelAdmin to show ADG admin only those applications which have been
+        submitted successfully.
+        """
+        qs = UserApplication.submitted_applications.all()
+
+        return qs
```

Comment:
Yes, Asad pointed it out earlier that it can be done with a single manager.

I used the qs variable to give better context. Anyways, have updated it.